lib/delta.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+var util = require('util');
+var deltaPrefix = '\u2206';
+var altDeltaPrefix = '\u0394';
+
+module.exports = {
+  deltaCounterName : function(metricName) {
+    if (module.exports.hasDeltaPrefix(metricName)) {
+      return metricName;
+    }
+    return deltaPrefix + metricName;
+  },
```

Comment:
You can probably write this as ternary expression
`hasDeltaPrefix(metricName) ? metricName : deltaPrefix + metricName`
Minor